webvtt.go

```diff
@@ -59,9 +59,23 @@ func ReadFromWebVTT(i io.Reader) (o *Subtitles, err error) {
 	var item = &Item{}
 	var blockName string
 	var comments []string
+
+	// For the Index Counter
+	saveIndex := true
+	var index int
+	var convErr error
+
 	for scanner.Scan() {
 		// Fetch line
 		line = strings.TrimSpace(scanner.Text())
+
+		// Fetch Index
+		if saveIndex {
+			index, convErr = strconv.Atoi(strings.Replace(line, "\ufeff", "", -1))
```

Comment:
You don't need to remove BOM header here since the first line of the file is ignored above

Reply:
fixed with https://github.com/asticode/go-astisub/pull/22/commits/4adf45bbc0604cf1200febf067bd524d1cb1e451